### ISSUE TITLE
run on mssql

### ISSUE THREAD

**Issue by rehman22** [ogomaemmanuel/VueChatApp#1]
can i run it on sql ..
good effort sir

**Comment by ogomaemmanuel** [comment]
Yes , You can run it on MySql. If you want to run it on SQL server, you will do a little modification

**Comment by ogomaemmanuel** [comment]
I have not been on it for sometime, But Everything should work well
